### DISCUSSION TITLE
Newbie question re HOOBs

### DISCUSSION THREAD

**Discussion by Bafrench1963** [thoukydides/homebridge-homeconnect#174] [General]
I am trying to make this work on my Hoobs device with no luck. Get a message in the log saying that my Homebridge is not the right version. Any thoughts?

**Comment by Bafrench1963** [comment]
Ignore … I found the note that this is a Homebridge plug-in (even if it says verified on Hoobs in Hoobs plug in market)

**Comment by thoukydides** [comment]
Correct, this is a Homebridge plugin, and despite its misleading name HOOBS is not Homebridge.

Other people have reported that there are problems with this plugin operating with HOOBS, and I have no plans to investigate or address them since I do not use HOOBS myself. I previously supported one of my plugins on HOOBS (homebridge-skybell) but it was significant extra work for something I never used.

I was not involved with the HOOBS team deciding to list this plugin on their site. If they want to list it then my opinion is that the HOOBS team are responsible for resolving any issues - whether that is by fixing HOOBS to actually be compatible with Homebridge, or by forking this plugin and maintaining whatever changes are necessary. I suggest you contact them: https://support.hoobs.org/ticket
